### PR TITLE
Acctivate the ngeo build by default

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by sbrunner** [comment]
Depends on https://github.com/camptocamp/ngeo/issues/286

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -510,6 +510,15 @@ $(APP_OUTPUT_DIR)/build.css: $(LESS_FILES) .build/node_modules.timestamp
...
 		.build/node_modules.timestamp
+	echo ==============================================
+	cat package.json
+	echo ==============================================
+	cat build.json
+	echo ==============================================
+	cat node_modules/ngeo/src/directives/map.js
+	echo ==============================================
+	ls -als node_modules/openlayers/src/**/*.js
+	echo ==============================================
```
Comment (elemoine) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: What is the reason for adding these echos?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Just for tests, I will remove the commit ...

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Removed :-)

**Comment by sbrunner** [comment]
@ochriste have a look on the pull request and be free to merge it :-)

**Comment by ochriste** [comment]
travis ok, as per sbrunner instruction, im merging

**Pull request merged**